docs/azure/index.yml

```diff
@@ -1,201 +1,231 @@
+          - text: Web API with CORS in Azure App Service
+            url: https://docs.microsoft.com/azure/app-service/app-service-web-tutorial-rest-api
+
+  - title: Build cloud native apps
+    linkLists:
+      - linkListType: quickstart
+        links:
+          - text: Run and debug a microservice in Kubernetes
+            url: https://docs.microsoft.com/azure/dev-spaces/quickstart-netcore-visualstudio
+      - linkListType: learn
+        links: 
+          - text: Create and deploy a cloud-native ASP.NET Core microservice
+            url: https://docs.microsoft.com/learn/modules/microservices-aspnet-core/
+      - linkListType: how-to-guide
+        links:
+          - text: Deploy a .NET Core app to Azure Container Registry
+            url: https://docs.microsoft.com/visualstudio/containers/hosting-web-apps-in-docker
+      - linkListType: tutorial
+        links:
+          - text: ASP.NET Core app with Visual Studio container tools
+            url: https://docs.microsoft.com/visualstudio/containers/container-tools
+          - text: Deploy and debug multiple containers in AKS
+            url: https://docs.microsoft.com/azure/dev-spaces/multi-service-netcore-visualstudio
+          - text: Dynamic configuration and feature flags using Azure App Config
+            url: https://docs.microsoft.com/azure/azure-app-configuration/use-feature-flags-dotnet-core
+
+  - title: Authentication and security
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Microsoft identity platform (Azure AD)
+            url: https://docs.microsoft.com/azure/active-directory/develop/
+      - linkListType: learn
+        links:
+          - text: Secure your application by using OpenID Connect and Azure AD
+            url: https://docs.microsoft.com/learn/modules/secure-app-with-oidc-and-azure-ad/
+          - text: Secure custom APIs with Microsoft Identity
+            url: https://docs.microsoft.com/learn/modules/identity-secure-custom-api/
+          - text: Secure an ASP.NET Core web app with the ASP.NET Identity framework
+            url: https://docs.microsoft.com/learn/modules/secure-aspnet-core-identity/
+      - linkListType: tutorial
+        links:
+          - text: Add sign-in to Microsoft to an ASP.NET web app
+            url: https://docs.microsoft.com/azure/active-directory/develop/tutorial-v2-asp-webapp
+          - text: Use Azure Key Vault with ASP.NET Core
+            url: https://docs.microsoft.com/azure/key-vault/tutorial-net-create-vault-azure-web-app
+          - text: Integrate Azure AD B2C with a web API
+            url: https://docs.microsoft.com/azure/active-directory-b2c/tutorial-single-page-app-webapi
+      - linkListType: reference
+        links:
+          - text: Azure Identity client library for .NET
+            url: https://docs.microsoft.com/dotnet/api/overview/azure/identity-readme?view=azure-dotnet
```

Comment:
@IEvangelist I have this link included twice. Originally I was just going to include it on the "Getting started" list in the first tile, but then I thought "Well what if they jump down to the Authentication tile looking for this?" The Authentication tile was intended to be Azure AD stuff.

What are your thoughts?

Reply:
Literally just now saw this, sorry for the delay. I think we should only include links once.